crypto/hash/hmac_ossl.c

```diff
@@ -240,7 +240,11 @@ static srtp_err_status_t srtp_hmac_init(void *statev,
         return srtp_err_status_auth_fail;
     }
 #else
-    if (HMAC_Init_ex(hmac->ctx, key, key_len, EVP_sha1(), NULL) == 0) {
+    if (key_len > INT_MAX) {
```

Comment:
Should this be moved above the `#ifdef`, since it is used in that code, too (line 238).

Reply:
The keyLen param of EVP_MAC_init is of type size_t so cast and check is not required.